src/utils/api/orderService/orderService.ts

```diff
@@ -0,0 +1,48 @@
+export interface OrderedMeal {
+    /** Meal's id */
+    id: string;
+
+    /** Meal's name */
+    name: string;
+
+    /** Meal's type */
+    price: number;
+
+    /** Number of units of the item ordered */
+    quantity: number;
+}
+
+export interface OrderState {
+    /** Unique identifier for the client */
```

Comment:
```suggestion
    /**
     * Unique identifier for the client who made the order.
     */
```
Когда мы смотрим на OrderState.userId важно понять какое отношение этот юзер имеет к этому заказу, и это отношение мы подчеркиваем в доках. Еще я бы сократил unique identifier до id, когда мы говорим об id это как бы по умолчанию подразумевается, что id уникальные у каждого объекта

Reply:
done